index.js

```diff
@@ -5,15 +5,18 @@ var express = require('express');
 var ParseServer = require('parse-server').ParseServer;
 var http = require('http');
 
-if (!process.env.DATABASE_URI) {
+var databaseUri = process.env.DATABASE_URI || process.env.MONGOLAB_URI
+
+if (!databaseUri) {
   console.log('DATABASE_URI not specified, falling back to localhost.');
 }
 
 var api = new ParseServer({
-  databaseURI: process.env.DATABASE_URI || 'mongodb://localhost:27017/dev',
+  databaseURI: databaseUri || 'mongodb://localhost:27017/dev',
   cloud: process.env.CLOUD_CODE_MAIN || __dirname + '/cloud/main.js',
-  appId: 'myAppId',
-  masterKey: 'myMasterKey'
+  appId: process.env.APP_ID || 'myAppId',
+  masterKey: process.env.MASTER_KEY || 'myMasterKey',
+  restAPIKey: process.env.REST_API_KEY || 'myRestAPIKey'
```

Comment:
restAPIKey shouldn't default to a value as it is optional and enforces keys on every request if set.